tools/ci_build/github/apple/assemble_apple_packaging_artifacts.sh

```diff
@@ -23,10 +23,36 @@ ORT_POD_VERSION=${4:?${USAGE_TEXT}}
 POD_ARCHIVE_BASENAME="pod-archive-${POD_NAME}-${ORT_POD_VERSION}.zip"
 PODSPEC_BASENAME="${POD_NAME}.podspec"
 
+
+# Macos requires a different structure for the framework
+# This will create the necessary symlinks for the macos framework before packaging
+# Adding the symlinks here rather than in the build script ensures that symlinks are not lost
```

Comment:
is it `xcodebuild -create-xcframework` which loses the symlinks?

Reply:
xcodebuild seem to be preserving link, but it seems to happen when assembling the pods